log-manager/src/main/java/io/airlift/log/Logging.java

```diff
@@ -128,12 +129,27 @@ public void setRootLevel(Level newLevel)
         setLevel(ROOT_LOGGER_NAME, newLevel);
     }
 
+    private void setBuiltInLevels()
```

Comment:
move it under the usage

Reply:
Please see the whole file, not just the diff. This was IMO the most logical place.